packages/hooks/src/__tests__/useCountdown.spec.tsx

```diff
@@ -120,9 +120,9 @@ describe('useCountdown', () => {
     });
 
     test('should count down from 3 to 0 after start is called and stop the counter at 1', async () => {
```

Comment:
```suggestion
    test('should count down from 4 to 0 after start is called and stop the counter at 1', async () => {
```
Not sure why this file has been changed tho 😅

Reply:
It was failing in one of the tests hence why I did it. Will try and undo and see whether it fails ornot.